### PR TITLE
Update ETC.tsv

### PR DESCRIPTION
**Pull request by Sunaries**
Minor fixes.

### CODE REVIEW

File: ETC.tsv [hunk 1]
```diff
@@ -1477,8 +1477,8 @@ ETC_20150317_001476	Team does not exist
 ETC_20150317_001477	{S18}{ol}{#ffffcc}This character does not exist
 ETC_20150317_001478	Cannot start game
 ETC_20150317_001479	{S18}{ol}{#ffffcc} Disconnected from the server due to verification failure
-ETC_20150317_001480	You have exceeded the maximum weight and cannot receive item. {nl}Sell or discard items in inventory to reduce weight.
-ETC_20150317_001481	You cannot receive the item. You have exceeded the maximum number of items allowed
+ETC_20150317_001480	You cannot obtain items anymore, because you have exceeded the maximum weight you can carry. {nl}Sell or discard items from your inventory to reduce weight.
```
Comment (laurencee) [inline ETC.tsv]: The comma before "because" is unnecessary:
"You cannot obtain items anymore because you have exceeded the maximum weight you can carry."

**Comment by Sunaries** [comment]
:+1: Thanks. Good to merge.

File: ETC.tsv [hunk 2]
```diff
@@ -1477,8 +1477,8 @@ ETC_20150317_001476	Team does not exist
...
+ETC_20150317_001481	You cannot obtain items anymore. You have exceeded the maximum number of items allowed.
```
Comment (Soukyuu) [inline ETC.tsv]: Should use the same wording for this and 1480 to make it look more uniform - only the reason changes. For example: "You cannot obtain any more items. You have exceeded the maximum number of items allowed."

File: ETC.tsv (hunk 1 above)
Comment (Soukyuu) [inline ETC.tsv]: Alternatively: "You cannot obtain any more items. You have exceeded the maximum weight you can carry."

**Comment by Sunaries** [comment]
Good to merge.

I've changed **ETC_20150317_001480** to just let the player know that they've exceeded their maximum carrying weight. Adding more text to the first sentence would only create a new line which just looks ugly on the screen.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**